core/src/apps/cardano/layout.py

```diff
@@ -210,6 +210,7 @@ async def confirm_sending(
     to: str,
     output_type: Literal["address", "change", "collateral-return"],
     network_id: int,
+    chunkify: bool = False,
```

Comment:
remove default value

Reply:
Done in [7e03759](https://github.com/trezor/trezor-firmware/pull/3237/commits/7e03759ea7c3f0e2e56218ee1bd5cb18bef9c134)